eclipse-che4z/che-che4z-lsp-for-cobol
# feat: Show diagnostic only for opened documents

## How Has This Been Tested?

1. Open a document with a copybook that contains an error
2. Ensure there are no errors for the copybook file
3. Open the copybook
4. Ensure that errors for copybooks appeared
5. Close the copybook file
6. Ensure errors for copybook disappeared

Also, all the following regressions should be tested:
1. Hover on a variable
2. Go to the definition
3. Autocompletion
4. List of actions for the variable/section, etc.
5. Open the same analyzed document for the second time (should be fast because all diagnostics and outline results are already here)

Also, it should fix #491

## Checklist:
